frontend/src/pages/AccountSend.vue

```diff
@@ -589,6 +595,20 @@ function useSendForm() {
         gasLimit: sendMax.value && sendingNativeToken ? sendMaxGasLimit : undefined,
       });
       void txNotify(tx.hash, ethersProvider);
+      if (viewingKeyPair.value?.privateKeyHex && userAddress.value && provider.value) {
```

Comment:
Is this `if` block necessary? I don't think we'd be able to make it this far if those values are undefined.

Reply:
That's a fair point against using `!`, we can leave as-is!